src/modules/variable/get.rs

```diff
@@ -61,29 +76,66 @@ impl SyntaxModule<ParserMetadata> for VariableGet {
 }
 
 impl TranslateModule for VariableGet {
+    #[allow(clippy::collapsible_else_if)]
```

Comment:
I did not want to collapse the `else if` chain here, because the two main branches under `Type::Array(_)` deal with the "pass by reference" and "pass by value" cases, and should therefore be at the same level.